deep_regression.py

```diff
@@ -0,0 +1,251 @@
+import pandas as pd
+from sklearn.utils import shuffle
+from sklearn.model_selection import train_test_split
+import tensorflow.contrib.slim as slim
+from tensorflow.contrib.framework.python.ops.variables \
+    import get_or_create_global_step
+from tensorflow.python.platform import tf_logging as logging
+
+
+'''
+Changeable parameters OR variables
+----------------------------------
+1. LEARNING_RATE
+2. reg_constant
+3. TRAINING_EPOCHS
+4. BATCH_SIZE
+5. DISPLAY_STEP
+6. test_size (splitting parameter for train:test)
+7. Hidden layer neurons number
+    N_HIDDEN_1
+    N_HIDDEN_2
+    N_HIDDEN_3
+    N_HIDDEN_4
+8. VARIANCE (for weights & biases, random normal initialization)
+9. tf.reduce_sum OR tf.reduce_mean
+10. Regularization method
+11. Gradient Descend Method
+    optimizer = tf.train.GradientDescentOptimizer()  ===> Gives all NaN values
+    optimizer = tf.train.AdamOptimizer()             ===> Gives definite values
+'''
+
+
+# =========== Parameters ============
+# LEARNING_RATE = 0.01                      # Learning rate
+BETA = 0.001                                # L2 regularization constant
+TRAINING_EPOCHS = 1000                       # EPOCHS
+BATCH_SIZE = 100                            # Set to 1 for whole sample at once
+DISPLAY_STEP = 100                          # Set to 1 for displaying all epoch
+DROPOUT_RATE = 0.9
+VARIANCE = 0.1                              # VARIANCE selection highly affects
+# Learning rate information and configuration (Up to you to experiment)
+INITIAL_LEARNING_RATE = 0.0002
+LEARNING_RATE_DECAY_FACTOR = 0.7
+NUM_OF_EPOCHS_BEFORE_DECAY = 1000
+
+# ============ Network Parameters ============
+HIDDEN_LAYERS = [500, 400, 600, 500, 500, 500]
+N_CLASSES = 1                       # Only one output prediction column
+
+
+def load_data_set(name=None):
+    """
+    This function reads the data file and extracts the features and labelled
+    values.
+    Then according to that patient is dead, only those observations would be
+    taken care
+    for deep learning trainig.
+  Args:
+    Nothing
+  Returns:
+    `Numpy array`, extracted feature columns and label column.
+  Example:
+    >>> read_dataset()
+    ( [[2.3, 2.4, 6.5],[2.3, 5.4,3.3]], [12, 82] )
+    """
+    data_feed = pd.read_csv('Brain_Integ_X.csv', skiprows=[0], header=None)
+    labels_feed = pd.read_csv('Brain_Integ_Y.csv', skiprows=[1], header=0)
+    survival = labels_feed['Survival']
+    censored = labels_feed['Censored']
+
+    survival = survival.values
+    censored = censored.values
+    data = data_feed.values
+    data = np.float32(data)
+
+    censored_survival = survival[censored == 1]
+    censored_data = data[censored == 1]
+
+    y = np.asarray(censored_survival)
+    x = np.asarray(censored_data)
+
+    print('Shape of X : ', x.shape)
+    print('Shape of Y : ', y.shape)
+    return (x, y)
+
+
+# ******************************************************************************
+
+
+with tf.Graph().as_default() as graph:
```

Comment:
This should be in a module named "train", while load_data should be in a module named "data_providers" or something.